src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/utils/constants.py

```diff
@@ -33,5 +33,10 @@
 OP_CONFORMANCE = "OP"
 API_CONFORMANCE = "API"
 
-REL_WEIGHTS_FILENAME = "rel_weights_REPLACE.lst"
+DEVICE_ARCHITECTURE_PROPERTY = "DEVICE_ARCHITECTURE"
+FULL_DEVICE_PROPERTY = "FULL_DEVICE_NAME"
+SUPPORTED_PROPERTIES = "SUPPORTED_PROPERTIES"
```

Comment:
we have such constants in Python API https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_runtime/test_properties.py#L149